### ISSUE TITLE
Qwen2-7B的输出用Qwen2-1.5B计算logp的时候报错

### ISSUE THREAD

**Issue by ZexuSun** [OpenRLHF/OpenRLHF#496]
output = self.model(sequences, attention_mask=attention_mask, position_ids=position_ids)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/deepspeed/utils/nvtx.py", line 18, in wrapped_fn
    ret_val = func(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/deepspeed/runtime/engine.py", line 1890, in forward
    loss = self.module(*inputs, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/transformers/models/qwen2/modeling_qwen2.py", line 1149, in forward
    outputs = self.model(
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/transformers/models/qwen2/modeling_qwen2.py", line 978, in forward
    inputs_embeds = self.embed_tokens(input_ids)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1553, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/modules/sparse.py", line 164, in forward
    return F.embedding(
  File "/root/miniconda3/envs/OpenRLHF/lib/python3.10/site-packages/torch/nn/functional.py", line 2267, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
    
    其中 sequences 和attention_mask是Qwen2-7B模型的输出， self.model是Qwen2-1.5B

**Comment by hijkzzz** [comment]
更新 HF 能否解决?

**Comment by catqaq** [comment]
请检查二者的词表是否是一致的？另外“ sequences 和attention_mask是Qwen2-7B模型的输出”具体是想做什么用途
